src/igvfd/schemas/curated_set.json

```diff
@@ -69,6 +69,17 @@
                 "transcriptome",
                 "variants"
             ],
+            "enum_descriptions": {
+                "barcodes": "Barcodes are lists of barcodes.",
+                "editing templates": "Editing templates are used in Saturation Genome Editing (SGE) assays.",
+                "elements": "Elements are lists of elements in the genome.",
```

Comment:
...lists of genomic elements to be assayed

Reply:
fixed